### PR TITLE
Update README and add .github/

### PR DESCRIPTION
**Pull request by PanoramicRum**
* Update README
* Add version badges to README
* Create .github/ file
*Add Contributing Guidelines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,62 +1,54 @@
...
+$ npm i --save opentok-acc-core
+```
 ```javascript
-const otCore = require('opentok-accelerator-core');
+const otCore = require('opentok-acc-core');
 ```
-Browser:
 ```javascript
-<script src="path/to/browser/opentok-accelerator-core.js"></script>
+<script src="path/to/browser/opentok-acc-core.js"></script>
```
Comment (marinaserranomontes) [inline README.md]: please note, the package is "opentok-accelerator-core"

File: README.md [hunk 2]
```diff
@@ -1,62 +1,54 @@
...
 
-```bash
-$ npm i --save opentok-accelerator-core
 ```
-webpack/Browserify:
+$ npm i --save opentok-acc-core
+```
 ```javascript
-const otCore = require('opentok-accelerator-core');
+const otCore = require('opentok-acc-core');
```
Comment (marinaserranomontes) [inline README.md]: please note, the package is "opentok-accelerator-core"

Comment (PanoramicRum) [inline README.md]: Fixed

File: README.md (hunk 1 above)
Comment (PanoramicRum) [inline README.md]: fixed, thanks

File: README.md [hunk 3]
```diff
@@ -1,62 +1,54 @@
...
+[![Build Status](https://travis-ci.org/opentok/accelerator-core-js.svg?branch=master)](https://travis-ci.org/opentok/accelerator-core-js)
+[![GitHub release](https://img.shields.io/github/release/opentok/accelerator-core-js.svg)](./README.md)
+[![license](https://img.shields.io/github/license/opentok/accelerator-core-js.svg)](./.github/CONTRIBUTING.md)
+[![npm](https://img.shields.io/npm/v/opentok-accelerator-core.svg)](https://www.npmjs.com/package/opentok-accelerator-core)
 
 ----------
 
 
-![logo](https://raw.githubusercontent.com/opentok/accelerator-core/master/tokbox-logo.png)
+![logo](https://raw.githubusercontent.com/opentok/accelerator-core-js/master/tokbox-logo.png)
```
Comment (marinaserranomontes) [inline README.md]: @PanoramicRum the logo should be before the status bar right?

Comment (PanoramicRum) [inline README.md]: We didn't define that but I also think it looks better that way. I changed it.

**Pull request merged**